morpho5/datastructures/matrix.c

```diff
@@ -949,6 +1042,106 @@ value Matrix_norm(vm *v, int nargs, value *args) {
     return MORPHO_FLOAT(matrix_norm(a));
 }
 
+/** Matrix eigensystem */
+bool Matrix_eigen(vm *v, objectmatrix *a, value *evals, value *evecs) {
```

Comment:
This is not a Morpho method. Should the M be lowercase?

Reply:
Good spot! I fixed the capitalization.